src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,75 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+
+class CalculatorTest {
+    @ParameterizedTest
+    @CsvSource(value = ["3,4", "1,4", "4,7"])
+    fun minus(a: Long, b: Long) {
+        assertThat(Calculator().minus(a, b) == a - b)
```

Comment:
검증되고 있지 않습니다. 아래와 같이 바꿔도 테스트를 통과합니다.
```suggestion
        assertThat(Calculator().minus(a, b) == a + b)
```